ws.py

```diff
@@ -197,6 +197,10 @@ def check_socket_connections():
         "room": GlobalVars.wrap.get_room("388"),
         "unwantedReasons": []
     },
+    {   "sites": ["meta.stackexchange.com"],
+        "room": GlobalVars.wrap.get_room("1121"),
```

Comment:
I *think* you will need to use `wrapm` rather than `wrap` to get the correct room.